tone-analyzer/v3.js

```diff
@@ -34,7 +34,7 @@ function ToneAnalyzerV3(options) {
 util.inherits(ToneAnalyzerV3, BaseService);
 ToneAnalyzerV3.prototype.name = 'tone_analyzer';
 ToneAnalyzerV3.prototype.version = 'v3';
-ToneAnalyzerV3.URL = 'https://gateway.watsonplatform.net/tone-analyzer/api';
+ToneAnalyzerV3.URL = 'https://gateway-d.watsonplatform.net/tone-analyzer/api';
```

Comment:
Don't change this here, instead set `url` in the object passed to the constructor or  set the `TONE_ANALYZER_URL` environment property. 

(You can also set `TONE_ANALYZER_USERNAME` and `TONE_ANALYZER_PASSWORD`, although the example wouldn't use them because of the hard-coded username and password).